nautobot/ipam/migrations/0020_delete_aggregate.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.16 on 2023-02-10 22:19
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("ipam", "0019_aggregate_to_prefix_data_migration"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
```

Comment:
While I think the app "blast radius" of removing Aggregate is likely much smaller than that of removing Region and Site, the same set of concerns apply - if an app has data models that depend on Aggregate, we need a documented and tested solution for how they can cleanly migrate to using Prefix in its place.

Reply:
As we had discussed yesterday, having a guide that explains using `run_before` in migrations would be the golden ticket.